charts/rancher-monitoring/v0.0.3/charts/prometheus/templates/_helpers.tpl

```diff
@@ -56,3 +56,51 @@
 {{- $name := include "app.name" . -}}
 {{- printf "%s-auth-%s" $name .Release.Name -}}
 {{- end -}}
+
```

Comment:
what do these changes do?

Reply:
This is to make sure we set namespace selector for rule and service monitor when installing project level monitoring.